src/main/java/com/artipie/asto/rx/RxTransactionWrapper.java

```diff
@@ -85,6 +85,13 @@ public Completable save(final Key key, final Flowable<ByteBuffer> content) {
         );
     }
 
+    @Override
+    public Completable move(final Key source, final Key destination) {
+        return CompletableInterop.fromFuture(
+            this.wrapped.move(source, destination)
+        );
+    }
```

Comment:
@olegmoz ``RxTrsansactionWrapper.move`` and ``RxStorageWrapper.move`` look identical, why?

Reply:
@amihaiemil that is true, but it is due to nature of delegation. Do not see any good solution to it, do you?

Reply:
@olegmoz Maybe we should open an Issue. This is clear duplicate code which smells. Anyway, not a topic for this PR. Thanks.